plugins/codemirror/plugin.js

```diff
@@ -36,6 +36,12 @@
 				instance.codeMirrorEditor.setValue(oldData);
 
 				var editableParent = editable.getParent();
+
+				var codeMirrorElement = editableParent.$.getElementsByClassName(
+					'CodeMirror'
+				);
```

Comment:
@fortunatomaldonado you can also get a reference to this element with

```js
instance.codeMirrorEditor.getWrapperElement();
```

Which returns the `<div>` element directly and avoids having to deal with an `HTMLCollection`
If you'd rather like a `CKEDITOR.dom.element` object, you can also use 

```js
editableParent.findOne('.CodeMirror');
```